GetStream/stream-chat-android
# [4395] `V6` Don't show link attachments in the attachment galleries

### 🎯 Goal

Implements #4395 

### 🛠 Implementation details

Filtered out attachments link attachments in both XML and Compose SDKs

### 🧪 Testing

Please repeat the test procedure on both apps:

#### Test Scheme:

1. Launch the app
2. Open a channel
3. Send a message that contains multiple media attachments (image and/or video) 
4. Open the message in the gallery
5. Exert all options such as reply, show in chat, save and delete
6. Exhaust delete until the last attachment is deleted

Repeat step 3 with the following variations:

- Message contains no text (should delete the message with the last attachment deleted)
- Message contains text (should not delete the message with the last attachment deleted)
- Message contains link (should not display the link attachment in the gallery and should not delete the message with the last attachment deleted)

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![gif](https://media1.giphy.com/media/NVBR6cLvUjV9C/giphy.gif?cid=ecf05e4783p0kgryzuyxyokxpjcedzo7q5vtq8vmerlons4c&rid=giphy.gif&ct=g)
